spec/features/pet_applications/new_spec.rb

```diff
@@ -134,7 +122,7 @@
       fill_in 'Address', with: '1241 W Bayaud Ave'
       fill_in 'City', with: 'Denver'
       fill_in 'State', with: 'CO'
-      # fill_in 'Zip', with: '80223'
+      # fill_in 'Zip', with: '80223' - this is the field that is skipped
```

Comment:
Thanks for leaving this in, I think it's helpful to let the next coder be able to see what we left.